src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -843,7 +843,7 @@ static void WindowEditorObjectiveOptionsMainPaint(rct_window* w, rct_drawpixelin
         {
             case OBJECTIVE_GUESTS_BY:
             case OBJECTIVE_GUESTS_AND_RATING:
-                stringId = STR_WINDOW_OBJECTIVE_VALUE_GUEST_COUNT;
+                stringId = STR_WINDOW_COLOUR_2_COMMA16;
```

Comment:
This seems a bit weird now, is this actually correct?

Reply:
Yeah, 3309 is a string that reads `{WINDOW_COLOUR_2}{COMMA16}`, and COMMA16 means “a 16-bit number formatted with thousands separators”.

Reply:
I see, the colour part irritated me a bit.